pages/webview/content.py

```diff
@@ -31,6 +32,10 @@ class Content(Page):
                           ' button[type="submit"][data-l10n-id="textbook-view-btn-get-this-book"]')
     )
     _ncy_locator = (By.CLASS_NAME, 'not-converted-yet')
+    _book_info_title_locator = (By.CSS_SELECTOR, '.booksContaining > div > span')
```

Comment:
Books containing is specific to a different page. If you need access to `booksContaining` then you must use `pages/content_page.py` which represents a single page all by itself w/o a book.